### PR TITLE
DEP-2215: update Saturday opening times for child benefit

### PR DESCRIPTION
**Pull request by lmugnai**

### CODE REVIEW

File: app/views/webchat/ChildBenefitView.scala.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h1 @if(!isIvrRedirect){class="header-margin-top"}>@messages("child.benefit.titl
     <div class="govuk-inset-text">
         <p>@messages("global.opening.times.title")</p>
         <p>@messages("global.opening.times.8to7_30")</p>
-        <p>@messages("global.opening.times.saturday.8to4")</p>
+        <p>@messages("global.opening.times.saturday.8to330")</p>
```
Comment (pratikrana06) [inline app/views/webchat/ChildBenefitView.scala.html:35]: for consistency across the messages file can put _ for 3:30 PM
"global.opening.times.saturday.8to3_30"

File: conf/messages [hunk 2]
```diff
@@ -18,6 +18,7 @@ global.opening.times.8to7_30=Monday to Friday, 8am to 7:30pm
 global.opening.times.8to4.includesat=Monday to Saturday, 8am to 4pm
 global.opening.times.mon.sat.8to7_30=Monday to Saturday, 8am to 7:30pm
 global.opening.times.saturday.8to4=Saturday, 8am to 4pm
+global.opening.times.saturday.8to330=Saturday, 8am to 3:30pm
```
Comment (pratikrana06) [inline conf/messages:21]: for consistency across the messages file can put _ for 3:30 PM
"global.opening.times.saturday.8to3_30"

**Review by pratikrana06 — CHANGES_REQUESTED** [review]
Added comment for _

File: conf/messages (hunk 2 above)
Comment (lmugnai) [reply conf/messages:21]: taking this out and creating child benefits own Saturday opening hours key as it has it's own view (not global)

**Review by pratikrana06 — APPROVED** [review]
Approved

**Pull request merged**